server/graphql/v2/mutation/VirtualCardMutations.ts

```diff
@@ -162,28 +162,31 @@ const virtualCardMutations = {
     type: new GraphQLNonNull(VirtualCard),
     args: {
       virtualCard: {
-        type: new GraphQLNonNull(VirtualCardUpdateInput),
-        description: 'Virtual Card being edited its new values',
+        type: new GraphQLNonNull(VirtualCardReferenceInput),
+        description: 'Virtual card reference',
+      },
```

Comment:
It's acceptable in Mutations (and maybe a good practice we would like to enforce) to use an `UpdateInput` instead of a `ReferenceInput`.

But that's also ok like this as we don't have a clear good practice established at this point.

Reply:
I agree if we have a generic update mutation but that not really the case here.